lib/core/src/Cardano/Wallet.hs

```diff
@@ -1541,8 +1541,17 @@ normalizeDelegationAddress s addr = do
 -- | A 'PartialTx' is an an unbalanced 'SealedTx' along with the necessary
 -- information to balance it.
 --
--- The 'inputs' and 'redeemers' must match the binary transaction contained in
--- the 'sealedTx'.
+-- The 'inputs' must match the binary transaction contained in
+-- the 'tx'.
+--
+-- The provided 'redeemers' will overwrite any redeemers inside the 'tx'. This
+-- is done as the internal redeemers in the 'tx' use an index referring to a
+-- 'TxIn', rather than a 'TxIn'. When we are adding extra inputs as part of
```

Comment:
```suggestion
-- is done as the internal redeemers in the 'tx' use an index referring to a
-- 'TxIn', rather than a 'TxIn' value. When we are adding extra inputs as
-- part of
```

Reply:
Went with `an actual 'TxIn'` instead